x-pack/legacy/plugins/code/public/components/commits/history.tsx

```diff
@@ -4,97 +4,27 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import {
-  EuiButton,
-  EuiFlexGroup,
-  EuiFlexItem,
-  EuiLoadingSpinner,
-  EuiPanel,
-  EuiText,
-  EuiTextColor,
-} from '@elastic/eui';
-import { FormattedMessage } from '@kbn/i18n/react';
-
 import _ from 'lodash';
 import moment from 'moment';
 import React from 'react';
 import { connect } from 'react-redux';
 import { i18n } from '@kbn/i18n';
+import { EuiButton, EuiFlexGroup, EuiFlexItem, EuiLoadingSpinner } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { CommitGroup } from './group';
 import { CommitInfo } from '../../../model/commit';
-import { CommitLink } from '../diff_page/commit_link';
```

Comment:
This component is now unused, but I was unsure whether it should be deleted or not.

Reply:
This is from diff page, I would say let's move the component to the new `commit` folder you created. @WangQianliang is doing some investigations on enabling the diff page, maybe he will need it.

Reply:
9e2387ec22